packages/frontend/src/components/ElectionForm/Races/useEditRace.tsx

```diff
@@ -50,7 +50,6 @@ export const useEditRace = (race, race_index) => {
     const validatePage = () => {
         let isValid = true
         let newErrors: any = {}
-        if (election.races.length > 1) {
```

Comment:
Me and Sam discussed removing this line. I believe this was a hold over from earlier UI iterations, and we opted to remove it since it was weakening the validation on the initial races that were created